docs/developing_iree/e2e_benchmarking.md

```diff
@@ -275,9 +275,24 @@ Android.
 The first two are to build it directly, either in a
 [`docker` container](https://www.tensorflow.org/lite/guide/build_android#set_up_build_environment_using_docker)
 or
-[in your own environment](https://www.tensorflow.org/lite/guide/build_android#set_up_build_environment_without_docker). Assuming you can build
-TensorFlow with Android, you can configure the TFLite `benchmark_model` binary
-in the following ways:
+[in your own
+environment](https://www.tensorflow.org/lite/guide/build_android#set_up_build_environment_without_docker).
+To build TensorFlow tools with Android:
+
+- Run `./configure` under TensorFlow repo.
+- Add below section to TensorFlow WORKSPACE file.
+
+```
+android_ndk_repository(
+    name="androidndk",
+    path="/full/path/to/android_ndk",
+)
+```
+
+XXX: Point the above instructions to somewhere outside IREE, e.g., TensorFlow.
```

Comment:
Use the
```
> Tip:<br>
> &nbsp;&nbsp;&nbsp;&nbsp;Point the above instructions ...
```
For better rendering?

The sentence here is a bit unclear to me. What do you mean by "point the above instructions"? You mean the NDK path?

Reply:
I mean the setup of building Tensorflow tools with Android. I think this should be documented somewhere, but I couldn't find it.

Reply:
Oh I see. One minor comment regarding the wording: s/Point/Place/ would be more clear to me.